nilearn/decoding/searchlight.py

```diff
@@ -249,6 +250,20 @@ class SearchLight(BaseEstimator):
         when y is supplied.
     %(verbose0)s
 
+    Attributes
+    ----------
+    scores_ : numpy.ndarray
+        3D array containing searchlight scores for each voxel, aligned
+         with the mask.
```

Comment:
```suggestion
         with the mask.

         .. versionadded:: 0.11.0
```

Can you also mention the version those attributes were added with a sphinx directive?

See above